pkg/controller/networkpolicy/crd_utils.go

```diff
@@ -61,6 +61,21 @@ func toAntreaServicesForCRD(npPorts []v1alpha1.NetworkPolicyPort, npProtocols []
 				ICMPCode: npProtocol.ICMP.ICMPCode,
 			})
 		}
+		if npProtocol.IGMP != nil {
+			curProtocol := controlplane.ProtocolIGMP
+			if npProtocol.IGMP.IGMPType != nil {
+				klog.V(4).InfoS("Get IGMP protocol information", "npProtocol.IGMP.IGMPType", *npProtocol.IGMP.IGMPType)
```

Comment:
The key word in log could be "IGMPType"?

Reply:
done